pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -616,10 +653,18 @@ def __init__(self, data, storage, payload, instclass):
 
         self._gather_mount_data_info()
 
+    def _is_dev_usable(self, dev):
+        maybe = not dev.protected and dev.size != Size(0)
+        if maybe and self.data.ignoredisk.onlyuse:
+            # all device's disks have to be in ignoredisk.onlyuse
+            maybe = set(self.data.ignoredisk.onlyuse).issuperset({d.name for d in dev.disks})
+
+        return maybe
```

Comment:
Isn't this code which should be in Blivet?

Reply:
Not really, it says which devices to show on this screen.